AirCasting/SessionViews/SessionCartView.swift

```diff
@@ -5,60 +5,69 @@
 //  Created by Lunar on 08/01/2021.
 //
 
-import SwiftUI
-import CoreLocation
-import CoreData
-import Charts
 import AirCastingStyling
+import Charts
+import CoreData
+import CoreLocation
+import SwiftUI
 
 struct SessionCartView: View {
-    
+    @State private var isFollowing = false
     @State private var isCollapsed = true
     @State private var selectedStream: MeasurementStreamEntity?
-    
+    @EnvironmentObject var persistenceController: PersistenceController
     @ObservedObject var session: SessionEntity
     let thresholds: [SensorThreshold]
-
+    
     var shouldShowValues: MeasurementPresentationStyle {
         let isFixed = session.type == .fixed
         let isDormant = session.type == .mobile && session.status == .FINISHED
         let shouldShow = isCollapsed && (isFixed || isDormant)
         
         return shouldShow ? .hideValues : .showValues
     }
+
     var showChart: Bool {
         !isCollapsed && session.type == .mobile && session.status == .RECORDING
     }
     
     var body: some View {
         VStack(alignment: .leading, spacing: 13) {
-            
             header
+            //            if let threshold = thresholdFor(selectedStream: selectedStream) {
+            StreamsView(selectedStream: $selectedStream,
+                        session: session,
+                        threshold: thresholds[0],
+                        measurementPresentationStyle: shouldShowValues)
             
-            if let threshold = thresholdFor(selectedStream: selectedStream) {
-                StreamsView(selectedStream: $selectedStream,
-                            session: session,
-                            threshold: threshold,
-                            measurementPresentationStyle: shouldShowValues)
-                
-                VStack(alignment: .trailing, spacing: 40) {
-                    if let selectedStream = selectedStream, showChart {
-                        pollutionChart(stream: selectedStream,
-                                       threshold: threshold)
-                    }
-                    if !isCollapsed {
-                        displayButtons(threshold: threshold)
-                    }
+            VStack(alignment: .trailing, spacing: 40) {
+                if let selectedStream = selectedStream, showChart {
+                    pollutionChart(stream: selectedStream,
+                                   threshold: thresholds[0])
```

Comment:
Same a above. 
I know you did that for mocking, for the future please add lots of comments&warnings to avoid situation like this when you missed hardcoded thingies.

Reply:
Of course, I misled myself - not the best